### PR TITLE
DAOS-11223 tests: More logging for rebuild/basic.py

### PR DESCRIPTION
**Pull request by liw**
The current logging is insufficient for diagnosing the problem.

Signed-off-by: Li Wei <wei.g.li@intel.com>
Required-githooks: true

### CODE REVIEW

File: src/tests/ftest/rebuild/basic.yaml
Comment (kccain) [inline src/tests/ftest/rebuild/basic.yaml:12]: LGTM especially with a fairly minimal DD_MASK setting. It may be worth running the test (with Test-tags to include both pr and these tests test_simple_rebiuld, test_multipool_rebuild) with and without the log_mask change to compare typical sizes of artifacts to make sure there may not be some unexpected consequences in the CI environment.

**Review by kccain — APPROVED** [review]

File: src/tests/ftest/rebuild/basic.yaml
Comment (liw) [reply src/tests/ftest/rebuild/basic.yaml:12]: Thanks, Ken. I missed the test-tag; I've added it in the new merge commit. Let's wait for the CI...

**Review by daltonbohning — COMMENTED** [review]
ftest LGTM, assuming the logs aren't too large, as already mentioned

**Comment by liw** [comment]
@kccain, @daltonbohning, would you consider the following acceptable?

Zipped daos_logs directories:

    liw2@m2 daos-11223 % ls -lh *.zip
    -rw-r--r--@ 1 liw2  staff   3.6M Sep 14 08:36 daos_logs-master.zip
    -rw-r--r--@ 1 liw2  staff    16M Sep 14 08:24 daos_logs-with_pr.zip

Unzipped daos_logs directory with bz2 files:

    liw2@m2 daos-11223 % du -Ah daos_logs-master
     26M	daos_logs-master
    liw2@m2 daos-11223 % du -Ah daos_logs-with_pr
    186M	daos_logs-with_pr

**Comment by liw** [comment]
F_H_L CI issue: CORCI-1226

**Comment by kccain** [comment]
> @kccain, @daltonbohning, would you consider the following acceptable?
> 
> Zipped daos_logs directories:
> 
> ```
> liw2@m2 daos-11223 % ls -lh *.zip
> -rw-r--r--@ 1 liw2  staff   3.6M Sep 14 08:36 daos_logs-master.zip
> -rw-r--r--@ 1 liw2  staff    16M Sep 14 08:24 daos_logs-with_pr.zip
> ```
> 
> Unzipped daos_logs directory with bz2 files:
> 
> ```
> liw2@m2 daos-11223 % du -Ah daos_logs-master
>  26M	daos_logs-master
> liw2@m2 daos-11223 % du -Ah daos_logs-with_pr
> 186M	daos_logs-with_pr
> ```

I think this could be acceptable. I used daos_test as a comparison point because its artifacts can be pretty large (the idea being we wouldn't want to approach or exceed its storage, as a worst-case criteria). I had to further compare daos_logs/ folder after bunzip2 *.bz2 files to be a little more certain. 

Before unzipping daos_logs/*.bz2 - master build [7412](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/master/7412/artifact/Functional%20Hardware%20Medium/daos_test/
) has about 100MB of storage across rebuild.py/daos_logs and suite.py/daos_logs

versus 186M daos_logs-with_pr, that on the surface looks worse when comparing to daos_test

But after bunzip2 *.bz2 the daos_test daos_logs/ directories added up to about 3G of storage consumed. Versus daos_logs-with_pr it seems to consume about 200MB.

**Comment by daltonbohning** [comment]
> @kccain, @daltonbohning, would you consider the following acceptable?
> ...
> Unzipped daos_logs directory with bz2 files:
> 
> ```
> liw2@m2 daos-11223 % du -Ah daos_logs-master
>  26M	daos_logs-master
> liw2@m2 daos-11223 % du -Ah daos_logs-with_pr
> 186M	daos_logs-with_pr
> ```

@kccain @liw I think it's mostly about the size of the archived logs, not necessarily zipped vs unzipped. master build [7412](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/master/7412/artifact/Functional%20Hardware%20Medium/daos_test/) has ~29M of archived logs in rebuild/basic.py/daos_logs/ and this PR has ~189M of archived logs in rebuild/basic.py/daos_logs/. 
Logs larger than 1M are individually compressed. I think most of the space consumption with this PR comes from many of the logs being a few hundred KB, but not large enough to be compressed. This might be acceptable - I'll check with @phender

**Comment by daltonbohning** [comment]
@liw I talked with @phender and this log size is okay

**Comment by liw** [comment]
@daltonbohning, @kccain, thank you for investigating the log size concern.

**Comment by liw** [comment]
@wangdi1, are you okay with these changes, please?

**Review by wangdi1 — APPROVED** [review]

**Comment by liw** [comment]
Thanks, Wang Di.

**Comment by ashleypittman** [comment]
Could the PR title be edited at landing please to clarify this only applies to a specific test.

**Comment by liw** [comment]
> Could the PR title be edited at landing please to clarify this only applies to a specific test.

Done.

**Pull request merged**
